specification/containerservice/resource-manager/Microsoft.ContainerService/aks/preview/2023-07-02-preview/examples/MachineGet.json

```diff
@@ -0,0 +1,34 @@
+{
+  "parameters": {
+    "api-version": "2023-07-02-preview",
+    "subscriptionId": "subid1",
+    "resourceGroupName": "rg1",
+    "resourceName": "clustername1",
+    "agentPoolName": "agentpool1",
+    "machineName": "aks-nodepool1-42263519-vmss00000t"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/dummyHostName",
```

Comment:
ID should include a fully specified resourceID, similar to the one below in `resourceId`